src/FixedDataTableCell.js

```diff
@@ -242,8 +242,8 @@ class FixedDataTableCell extends React.Component {
           style={columnResizerStyle}
           onMouseDown={this._onColumnResizerMouseDown}
           onTouchStart={this.props.touchEnabled ? this._onColumnResizerMouseDown : null}
-          onTouchEnd={this.props.touchEnabled ? e => e.stopPropagation() : null}
-          onTouchMove={this.props.touchEnabled ? e => e.stopPropagation() : null}>
+          onTouchEnd={this.props.touchEnabled ? _suppressEvent : null}
```

Comment:
#280 
Not sure if the changes are necessary here. I couldn't test it. For now we'll keep the changes here, and revert them if not needed.

Reply:
Doesn't it need to be this._suppressEvent?

@davidhaimson heads up about this.  You may want to test on v1 beta once this is merged.

Reply:
My bad, it does need `this`. 
Will fix